uPortal-layout/src/main/java/org/apereo/portal/layout/LayoutPortlet.java

```diff
@@ -12,7 +12,7 @@
  * express or implied. See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apereo.portal.rest.layout;
+package org.apereo.portal.layout;
```

Comment:
For the initial moves, I have been keeping all classes in their original packages... even if it means a single package exists in multiple modules.  I'm just trying to "limit the variables" in a process that's already complicated.

We may soon decide to repackage everything;  perhaps we will decide that all classes within a module like `:uPortal-security:uPortal-security-xslt` should exist in `org.apereo.portal.security.xslt` (and subpackages).  Thbis can be a topic for the conference.